scripts/install_authz_dependencies.sh

```diff
@@ -9,26 +9,32 @@ PROJECT_ROOT=$(realpath ${SOURCE}/..)
 
 source "${PROJECT_ROOT}/scripts/dependency_install_functions.sh"
 
-# This script will install all of the dependencies needed by DataFed 1.0
-sudo apt-get update
-sudo dpkg --configure -a
-sudo apt-get install -y libtool build-essential g++ gcc libboost-all-dev \
-autoconf automake make git python3-pkg-resources python3-pip pkg-config \
-libglobus-common-dev wget libssl-dev
-sudo apt-get install -y libzmq3-dev 
-
-install_cmake
-cd ~
-
-python3 -m pip install --upgrade pip
-python3 -m pip install setuptools
-
-install_protobuf
-cd ~
-
-install_libsodium
-cd ~
-
-install_libzmq
-cd ~
-
+packages=("libtool" "build-essential" "g++" "gcc" "libboost-all-dev" "autoconf" "automake" "make" "git" "python3-pkg-resources" "python3-pip" "pkg-config" "libglobus-common-dev" "wget" "libssl-dev" "libzmq3-dev")
+externals=("cmake" "protobuf" "libsodium" "libzmq")
+
+if [ $# -eq 1 ]; then
+  case "$1" in
+    -h|--help)
+      # If -h or --help is provided, print help
+      echo "Usage: $0 [-h|--help] [unify]"
+      ;;
+    unify)
+      # If 'unify' is provided, print the packages
+      echo -n "${packages[@]}" >> "$apt_file_path"
+      echo -n "${externals[@]}" >> "$ext_file_path"
```

Comment:
You need to add a space in here at the end when you are appending to the same line in apt_file_path it is not leaving a space at the end. So the last and first packages are getting munged together i.e. libzm3-devlibtool. Make sure you also add a comment as to why the space is necessary.

"${packages[@]} "